src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -45,6 +46,22 @@ struct Unit<position<T_Type> >
         return unit;
     }
 };
+template<typename T_Type>
+struct UnitDimension<position<T_Type> >
+{
+    static std::vector<float_64> get()
+    {
+        /* L, M, T, I, theta, N, J
+         *
+         * position is in meter: m
+         *   -> L
+         */
+        std::vector<float_64> unitDimension( 7, 0.0 );
```

Comment:
Could  you define a const variable 
```
BOOST_CONSTEXPR_OR_CONST uint_32 NUnitDimensions = 7;
```
and use `NUnitDimensions` instead of a *magic* `7` defined in another file `UnitDimension.hpp` or the OpenPMD repo?
```
 std::vector<float_64> unitDimension( NUnitDimensions, 0.0 ); 
```